src/clj/puppetlabs/services/master/master_core.clj

```diff
@@ -896,7 +896,8 @@
    (schema/optional-key "job_id") schema/Str
    (schema/optional-key "transaction_uuid") schema/Str
    (schema/optional-key "options") {(schema/optional-key "capture_logs") schema/Bool
-                                    (schema/optional-key "log_level") (schema/enum "debug" "info" "warning" "error")}})
+                                    (schema/optional-key "log_level") (schema/enum "debug" "info" "warning" "error")
+                                    (schema/optional-key "boltlib") schema/Str}})
```

Comment:
Should perhaps allow array as well.

Reply:
What does the Orchestrator send?

Reply:
https://github.com/puppetlabs/orchestrator/pull/1338/files#diff-e3cb1255de6ee2d1bf0cf60785425dd0R310

I updated orchestrator to send two options, the first is whether or not to load bold (we could probably just interpret the presence of boltlib_path for that) and the path to the boltlib module code on disk. Allowing only a list type seems best even if we only ever have a single element.